connectors/websocket/pom.xml

```diff
@@ -27,10 +27,10 @@
     <version>1.2.0-SNAPSHOT</version>
   </parent>
 
-  <artifactId>edgent-connectors-websocket-wsclient</artifactId>
```

Comment:
unrelated to this change I see that j8 and j7 websocket/pom.xml declare a dependency on the direct provider.  That can't be correct. A connector can't depend on a provider.